clients/js/src/generated/index.ts

```diff
@@ -9,6 +9,5 @@
 export * from './accounts';
 export * from './errors';
 export * from './instructions';
-export * from './pdas';
```

Comment:
I can't comment on the `generate-clients.mjs` file, but we are missing a rename there. This is why the `pdas.ts` file is not being generated.

https://github.com/paladin-bladesmith/stake-program/blob/62b2dc6dce04b520e411d2e2fa82907eacdb0b76/scripts/generate-clients.mjs#L26

```
"[programNode]stake" -> "[programNode]paladinStakeProgram"